src/app/resources/rate-resources/resources-rate.component.html

```diff
@@ -0,0 +1,39 @@
+<h1 i18n>Submit your rating</h1>
+<form [formGroup]="ratingForm" (ngSubmit)="onSubmit()">
+    <div hidden>
+        <mat-form-field>
+            <input matInput i18n-placeholder [(ngModel)]="this.id" type="text" formControlName="id">
+            <mat-error><planet-form-error-messages [control]="ratingForm.controls.id"></planet-form-error-messages></mat-error>
+        </mat-form-field>
+    </div>
+    <div hidden>
+        <mat-form-field>
+            <input matInput i18n-placeholder [(ngModel)]="this.user" type="text" formControlName="user">
+            <mat-error><planet-form-error-messages [control]="ratingForm.controls.user"></planet-form-error-messages></mat-error>
+        </mat-form-field>
+    </div>
+    <div>
+        <mat-radio-group>
+            <label i18n>Gender:</label>
+            <mat-radio-button value="true" name="genderToggler" (change)="isMale(true)" i18n>Male</mat-radio-button>
```

Comment:
this is not to be input by user instead it should be taken by default based on logged in user